### PR TITLE
add french keywords to Skill

### LINKED ISSUE #3413: Add french keywords to Skill model
# Description
Skills currently have Names and Descriptions in French and English, but only a single array of keywords. Split this into two: an array of English keywords and an array of French ones. Make this change to the backend, the API, and modify the admin Skill forms and tables to match.

## Acceptance Criteria
- [x] Modify the graphql schema so that Skill.keywords is a new type with definition `{en: [String!], fr: [String!]}`
- [x] Modify the graphql schema so Skill.keywords can be set properly when creating or updating a skill
- [x] Modify the Skill.php model and the skills table (with a migration) so that localized keywords can be stored
- [x] Modify SkillFactory to generate one random set of keywords, and store it in en and fr with " EN" and " FR" suffixes
- [x] Modify fakeSkills.ts to generate skills in the same way
- [x] Update SkillTable to only show keywords in the current language
- [x] Update CreateSkill.tsx and UpdateSkill.tsx to show two keywords fields, one for each language, and save them properly

### PR DESCRIPTION
**Pull request by vd1992**
closes #3413 

Changes Skill to have its keywords in EN and FR from the front to backend 
To test: play around, make sure you can create/update skills properly along with exploring various corners to watch for accidents since this issue touches a broad area

I did a separate commit to run prettier, given I was touching a bunch of GraphQL files I figured cleaning them up would be good, not much changed in that commit just spacing mainly

### CODE REVIEW

**Review by JamesHuf — COMMENTED** [review]
This looks really great! I do have one potential concern below though. Also I'm curious whether we'd be like to prevent duplicate keywords or not.

File: api/database/migrations/2022_07_20_214048_update_skills_french_keywords.php [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+{
+    /**
+     * Run the migrations.
+     *
+     * @return void
+     */
+    public function up()
+    {
+        Schema::table('skills', function (Blueprint $table) {
+            $table->dropColumn('keywords');
```
[the remark this answers is not in the document]
Comment (vd1992) [reply api/database/migrations/2022_07_20_214048_update_skills_french_keywords.php:17]: I tried the change() method at first, but you apparently can't modify `jsonb` https://laravel.com/docs/8.x/migrations#updating-column-attributes
I'll need help if something more complicated is needed here

[the remark this answers is not in the document]
Comment (vd1992) [reply api/database/migrations/2022_07_20_214048_update_skills_french_keywords.php:17]: Okay @JamesHuf, that was resolved as not being a concern

**Review by JamesHuf — APPROVED** [review]
Should be good to go!

**Pull request merged**